environments/.mariadb104.sample.env

```diff
@@ -0,0 +1,4 @@
+MARIADB_PORT=3306
```

Comment:
For the sake of simplicity, let's provide a single `.mariadb.sample.env` file. If the user needs multiple variations they can copy those as non-sample files and update their workspace.json config. We should also include a README about how to use these environment files specifically for mariaDB.

Reply:
I have added Readme files and consolidated the environment files